### PR TITLE
feat: images and layout changes for cards

### LINKED ISSUE #280: Missing Program Mode Icon assets (image resources)
**Describe the issue**

Missing Program Mode Icon assets (image resources). 

Make a PR with the 4 image resources in the drawable folder.
These images are required in the launcher screen.

**Would you like to work on the issue?**
@jddeep @adityastic

### PR DESCRIPTION
**Pull request by adityastic**
Fixes #280 

**Changes**:
- Including Images and Layout Changes

@jddeep @suresh-gandhi @CloudyPadmal

### CODE REVIEW

**Comment by adityastic** [comment]
included svgs as well, good luck

**Review by jddeep — APPROVED** [review]

**Review by jddeep — COMMENTED** [review]
@adityastic Please check, the build is failing.

File: app/src/main/res/layout/card_relax.xml [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:layout_centerInParent="true"
+            android:orientation="horizontal"
+            android:padding="16.0dip">
+
+            <ImageView
+                android:layout_width="match_parent"
+                android:layout_height="match_parent"
+                android:layout_marginStart="8dp"
```
Comment (suresh-gandhi) [inline app/src/main/res/layout/card_relax.xml:36]: @adityastic just add the references in dimen file. Also everywhere else where hardcoded. Rest looks great 👍

**Review by suresh-gandhi — CHANGES_REQUESTED** [review]

File: app/src/main/res/layout/card_relax.xml (hunk 1 above)
Comment (jddeep) [reply app/src/main/res/layout/card_relax.xml:36]: @suresh-gandhi No worries. I will be taking care of these as I also have to see how they fit in our actual Activity and tweak accordingly.

**Comment by mariobehling** [comment]
Please always add screenshots in future.

**Pull request merged**